boards/frdm-kw41z/board.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     board_frdm-kw41z
+ * @{
+ *
+ * @file
+ * @brief       Board specific initialization for the FRDM-KW41Z
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "periph/gpio.h"
+#include "periph/rtt.h"
+
+static inline void set_lpuart_clock_source(void)
+{
+    /* Use MCGIRCLK (internal reference 4 MHz clock) */
+    SIM->SOPT2 = (SIM->SOPT2 & ~SIM_SOPT2_LPUART0SRC_MASK) | SIM_SOPT2_LPUART0SRC(3);
+}
+
+void board_init(void)
+{
+    /* initialize the CPU core */
+    cpu_init();
+
+    set_lpuart_clock_source();
+#if MODULE_XTIMER
+    /* Start the RTT, used as time base for xtimer */
+    rtt_init();
+#endif
```

Comment:
RTT cannot be selected independently of xtimer needs? Besides, this is only required for this board? I tend to think that this is more a CPU requirement, but I can be mistaken.

Reply:
It is because this board uses the lptmr hardware timer for xtimer, while the other boards use the PIT hardware timer for xtimer. the LPTMR driver implementation uses RTT for a time base reference. I would like to improve the lptmr driver, but limitations in the hardware in combination with requirements from the xtimer implementation makes it difficult to make a good implementation without a separate time reference right now.
If you like I can change the default config to use PIT like the other kinetis boards